Resources/views/Collector/db.html.twig

```diff
@@ -98,289 +107,340 @@
+                        {% endif %}
+                    {% endif %}
+                {% endif %}
             </div>
-        {% else %}
-            <div class="metrics">
-                <div class="metric">
-                    <span class="value">{{ collector.cacheCounts.hits }}</span>
-                    <span class="label">Hits</span>
-                </div>
-
-                <div class="metric">
-                    <span class="value">{{ collector.cacheCounts.misses }}</span>
-                    <span class="label">Misses</span>
-                </div>
+        </div>
 
-                <div class="metric">
-                    <span class="value">{{ collector.cacheCounts.puts }}</span>
-                    <span class="label">Puts</span>
-                </div>
+        <div class="tab">
+            <h3 class="tab-title">Connections &amp; EM</h3>
+
+            <div class="tab-content">
+                <h3>Database Connections</h3>
+
+                {% if not collector.connections %}
+                    <div class="empty empty-panel">
+                        <p>There are no configured database connections.</p>
+                    </div>
+                {% else %}
+                    {{ helper.render_simple_table('Name', 'Service', collector.connections) }}
+                {% endif %}
+
+                <h3>Entity Managers</h3>
+
+                {% if not collector.managers %}
+                    <div class="empty empty-panel">
+                        <p>There are no configured entity managers.</p>
+                    </div>
+                {% else %}
+                    {{ helper.render_simple_table('Name', 'Service', collector.managers) }}
+                {% endif %}
             </div>
-
-            {% if collector.cacheRegions.hits %}
-                <h3>Number of cache hits</h3>
-                {{ helper.render_simple_table('Region', 'Hits', collector.cacheRegions.hits) }}
-            {% endif %}
-
-            {% if collector.cacheRegions.misses %}
-                <h3>Number of cache misses</h3>
-                {{ helper.render_simple_table('Region', 'Misses', collector.cacheRegions.misses) }}
-            {% endif %}
-
-            {% if collector.cacheRegions.puts %}
-                <h3>Number of cache puts</h3>
-                {{ helper.render_simple_table('Region', 'Puts', collector.cacheRegions.puts) }}
-            {% endif %}
-        {% endif %}
-    {% endif %}
-
-    {% if collector.entities|length > 0 %}
-        <h2>Entities Mapping</h2>
-
-        {% for manager, classes in collector.entities %}
-            {% if collector.managers|length > 1 %}
-                <h3>{{ manager }} <small>entity manager</small></h3>
-            {% endif %}
-
-            {% if classes is empty %}
-                <div class="empty">
-                    <p>No loaded entities.</p>
-                </div>
-            {% else %}
-                <table>
```

Comment:
Not sure why are we changing position of this section, this makes the PR quite hard to review